packages/info/configParser.ts

```diff
@@ -0,0 +1,37 @@
+import * as path from "path";
+import chalk from "chalk";
+import * as prettyjson from "prettyjson";
+
+export function getNameFromPath(fullPath: string): string {
+	const filename = fullPath.replace(/^.*[\\\/]/, "");
+	return filename;
+}
+
+export function resolveFilePath(relativeFilePath: string) {
+	const configPath = path.resolve(process.cwd() + "/" + relativeFilePath);
+	return configPath;
+}
+
+export function fetchConfig(configPath: string) {
+	let config = null;
+	config = require(configPath);
+	try {
+	} catch (e) {
+		process.stdout.write(chalk.red(`Error:`, e.code) + "\n");
```

Comment:
can you stick either with the template string or normal string concatenation rather than mixing them both?

Reply:
Ok